### ISSUE TITLE
"Angle system"?

### ISSUE THREAD

**Issue by draialexis** [skiasaurus/tau_manifesto#5]
"**Perhaps the most elementary angle system is degrees**"

This clause feels weird to me, in both versions. Would you talk about an '_angle system_' naturally?

Could it be a typo where the author meant '_angle measurement system_', or are those two actually equivalent?

I'm kinda scared to touch it before I understand the source's choice better, and my research hasn't turned up much.

PS: this is a category of issues that may be useful during the whole proofing process, if you're ok with it being here.

**Comment by skiasaurus** [comment]
> PS: this is a category of issues that may be useful during the whole proofing process, if you're ok with it being here.

I'm actually beginning to think issues might be a better place than commit comments for _all_ of our discussions, because they're easier to track over the long run and can be linked to multiple commits to represent different versions of a passage. Also, they can function as a checklist: once all (or nearly all) of the issues have been resolved, we'll know we're ready to publish.

That said, rather than putting several different things in one issue, let's keep them in separate issues and group them together using a label to represent the "category" (by which I assume you mean places where you want to better understand the English usage before making a translation choice).

> Could it be a typo where the author meant '_angle measurement system_', or are those two actually equivalent?

"Angle system" can definitely be taken to mean "angle measurement system," so feel free to translate it as such.

**Comment by skiasaurus** [comment]
By the way, if you make a commit that solves a GitHub issue, just use one of these words

- close
- closes
- closed
- fix
- fixes
- fixed
- resolve
- resolves
- resolved

followed by "#x" (where x is the issue number) somewhere in your commit message, and GitHub should automatically mark the issue as resolved.

**Comment by draialexis** [comment]
Thanks for the Git tips, I thought I remembered something like that. That's cool.

> "Angle system" can definitely be taken to mean "angle measurement system," regardless of whether his omission of the word "measurement" is a typo or deliberate; so feel free to translate it as such.

And thanks for that, I'll act accordingly.

> I'm actually beginning to think issues might be a better place than commit comments for all of our discussions

That would work for me too.

>  let's keep them in separate issues and group them together using a label

I agree. I created a "clarification" label to start with.

If that's ok, I'll keep the current discussions in the comments where they are, but I won't create new ones for that purpose.

Thanks again!

**Comment by draialexis** [comment]
Hmm seems like Git will only mark the issue as resolved once we've merged this branch with master. Should we schedule daily merges, or should we just close issues manually for now?

**Comment by skiasaurus** [comment]
>Hmm seems like Git will only mark the issue as resolved once we've merged this branch with master.

Whoops, missed that.

In an ideal world, we'd merge periodically; but the thing about doing a merge in our current state is that if I were to checkout `master` and run `git merge alex` right now, it would merge _all_ the changes that have been made on `alex`, including the ones we haven't finished discussing. The solution to this going forward (if we did want to merge as we went) would be to create more branches for separate fixes that can be merged individually later on, as in (for example) the [git-flow model](https://nvie.com/posts/a-successful-git-branching-model/).

I don't know about you, but I've never actually used or seen this workflow in action, only read about it; so I would have to spend some time learning more about it. But there would also be benefits to making that time investment. So the question is: is it worth it?

**Comment by draialexis** [comment]
That seems a little more involved than it needs to be honestly. I vote for closing issues manually. I've worked with git-flow in the past a little and it's useful, but I don't really think that this model would apply to our situation. Plus, it was a while ago, I'm a bit blurry.

Related point: I guess we can schedule a merge for the next time I won't be able to work on this, which is Friday 28, roughly all day. Specifically, I won't work on the translation between 4AM on 08/28 and 3PM on 08/29 (your time, 10PM on 08/27 and 9AM on 08/29, if I got that part right)

We can ensure that the branch is stable and issues are settled before 6PM EDT on 08/27 -- midnight CEST on 08/28. Then we merge.

Sounds good?
